src/ontology/uberon-edit.obo

```diff
@@ -224519,6 +224519,16 @@ is_a: UBERON:7500094 ! tubercle of calcaneus
 property_value: http://purl.org/dc/elements/1.1/date 2021-08-04T00:27:56Z xsd:dateTime
 property_value: term:editor http://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500101
+name: antorbital notch
+def: "The antorbital notch is a prominence of bone located anterolaterally of the antorbital fenestra." [doi:10.5479/si.00810282.627]
+is_a: UBERON:0006846 ! surface groove
+relationship: part_of UBERON:0003129 ! skull
+property_value: present_in_taxon "Cetacea" xsd:string
+property_value: present_in_taxon "Equus" xsd:string
```

Comment:
These should link to the terms in NCBI_Taxon, rather than being strings.

Reply:
@dosumis I don't see an option for that in the annotation properties.

Reply:
@dosumis how do I do that?

Reply:
Just realised that Protege doesn't make it easy as you can't use the class expression editor for an AP value.
- you can  navigate to the term by hand like this - which is not very practical
![image](https://user-images.githubusercontent.com/112839/149385593-e7ec53c0-f7e5-492c-8aa5-50cb22803fef.png)
- much easier is to search for the term 
![image](https://user-images.githubusercontent.com/112839/149385793-b35333b0-e929-4f17-b30e-e4bfe052fd08.png)
copy the IRI (use COMMAND U to display) 
![image](https://user-images.githubusercontent.com/112839/149385929-2cb3c764-f2bd-400f-af99-25a55bc9ce4f.png)
then paste into the IRI editor on the term you're editing:
![image](https://user-images.githubusercontent.com/112839/149386081-794c2070-b56e-49e6-9881-91c887b454ec.png)
=>
![image](https://user-images.githubusercontent.com/112839/149386112-d191a604-e9e5-42a5-8868-bb7ebf505add.png)